src/pages/VenuePage/VenuePage.tsx

```diff
@@ -239,6 +241,15 @@ const VenuePage = () => {
       : undefined
   );
 
+  useEffect(() => {
+    if (user && profile && venue && venueId) {
+      mixpanel.track()("VenuePage loaded", {
+        venueId: venue.id,
+        template: venue.template,
+      });
+    }
+  }, [user, profile, venue, venueId, mixpanel]);
+
```

Comment:
Only pass through the bits of venue we care about, so it doesn't break memo on unrelated things:

```suggestion
  const venueTemplate = venue.template;
  
  useEffect(() => {
    if (user && profile && venueId && venueTemplate) {
      mixpanel.track()("VenuePage loaded", {
        venueId,
        template: venueTemplate,
      });
    }
  }, [user, profile, venueId, venueTemplate, mixpanel]);
```